canary/manifest/cpx-exporter.yaml

```diff
@@ -1,68 +1,66 @@
-apiVersion: extensions/v1beta1

```

Comment:
Can we make changes in all the exporter file as per the example here : https://github.com/citrix/citrix-k8s-ingress-controller/tree/master/metrics-visualizer#configuring-citrix-adc-metrics-exporter
Looks, like these canary and other examples were not updated. Let me know if case of any doubt.